v2/pkg/release/local_stored_collector.go

```diff
@@ -105,10 +107,15 @@ func (o *LocalStorageCollector) ReleaseImageCollector(ctx context.Context) ([]v1
 			if len(oci.Manifests) == 0 {
 				return []v1alpha3.CopyImageSchema{}, fmt.Errorf(errMsg, "image index not found ")
 			}
-			manifest := strings.Split(oci.Manifests[0].Digest, ":")[1]
+			validDigest, err := digest.Parse(oci.Manifests[0].Digest)
+			if err != nil {
+				return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[ReleaseImageCollector] invalid digest for image index %s: %v", oci.Manifests[0].Digest, err)
+			}
+
+			manifest := validDigest.Hex()
```

Comment:
Hi @sherine-k,

It seems the Hex() func is deprecated. It is saying to use Digest.Encoded instead